src/styles/box.scss

```diff
@@ -72,18 +72,26 @@
 
 	&__info {
 		margin-bottom: 4px;
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
```

Comment:
Let's add here 'gap: 10px' and replace padding in __body on 'padding: 10px 10px 0 10px;'